src/barril/units/unit_database.py

```diff
@@ -887,7 +902,7 @@ def GetUnitNames(self, quantity_type):
         """
         return [x.name for x in self.GetInfos(quantity_type)]
 
-    def GetInfo(self, quantity_type, unit, fix_unknown=False):
+    def GetInfo(self, quantity_type, unit, fix_unknown=False, fix_legacy=False):
```

Comment:
I think the default should be True

```suggestion
    def GetInfo(self, quantity_type, unit, fix_unknown=False, fix_legacy=True):
```